apps/demo/src/app/cart/components/add-to-cart-notification/components/add-to-cart-notification/add-to-cart-notification.component.html

```diff
@@ -1,21 +1,20 @@
-<demo-modal-portal>
-  <daff-modal [show]="open$ | async" (hide)="onHide()" [verticalPosition]="_verticalPosition"
-    [horizontalPosition]="_horizontalPosition">
-    <div class="demo-add-to-cart-notification">
-      <div class="demo-add-to-cart-notification__header"  *ngIf="!(loading$ | async)">
-        <span class="demo-add-to-cart-notification__check-icon"><fa-icon [icon]="faCheck" aria-hidden="true"></fa-icon></span>
-        <span>Added to Cart</span>
-        <fa-icon [icon]="faTimes" aria-hidden="true" (click)="onHide()" class="demo-add-to-cart-notification__close"></fa-icon>
-      </div>
-      <demo-product-added *ngIf="!(loading$ | async)" class="demo-add-to-cart-notification__product-added"
-        [product]="product$ | async" [qty]="productQty$ | async"></demo-product-added>
-      <div *ngIf="!(loading$ | async)" daff-button-set class="demo-add-to-cart-notification__button-set">
-        <button type="button" daff-stroked-button (click)="onHide()" demoViewCart>
-          View Cart ({{(cartItemCount$ | async)}})
-        </button>
-        <button type="button" daff-button color="secondary" (click)="onHide()" demoProceedToCheckout>Checkout</button>
-      </div>
-      <daff-loading-icon *ngIf="(loading$ | async)"></daff-loading-icon>
-    </div>
-  </daff-modal>
-</demo-modal-portal>
+<div class="demo-add-to-cart-notification">
+  <div class="demo-add-to-cart-notification__header"  *ngIf="!(loading$ | async)">
+    <span class="demo-add-to-cart-notification__check-icon">
+      <fa-icon [icon]="faCheck" aria-hidden="true"></fa-icon>
+    </span>
+    <span>Added to Cart</span>
+    <button daff-icon-button class="demo-add-to-cart-notification__close" (click)="onHide()">
+      <fa-icon [icon]="faTimes" aria-hidden="true"></fa-icon>
+    </button>
+  </div>
+  <demo-product-added *ngIf="!(loading$ | async)" class="demo-add-to-cart-notification__product-added"
+    [product]="product$ | async" [qty]="productQty$ | async"></demo-product-added>
+  <div *ngIf="!(loading$ | async)" daff-button-set class="demo-add-to-cart-notification__button-set">
+    <button type="button" daff-stroked-button demoViewCart>
+      View Cart ({{(cartItemCount$ | async)}})
+    </button>
+    <button type="button" daff-button color="secondary" demoProceedToCheckout>Checkout</button>
```

Comment:
clicking either of these buttons should also close the modal.

Reply:
I plan on addressing this in the next PR. See my note in the PR description:

> @lderrickable @xelaint This will have subsequent PRs to restore pre-existing functionality, like closing the modal. Additionally, it will follow up with better action hygiene. I didn't want this PR to become unmanageable so, I did the smallest amount of code I could to upgrade.